### DISCUSSION TITLE
update values in runtime

### DISCUSSION THREAD

**Discussion by gals-ma** [sse-secure-systems/connaisseur#354] [Q&A] [answered]
Hi,

Is there a way I can update values in runtime (i.e upgrade) rather than uninstalling + installing the helm?
For example, changing the policy or edit the detection flag.

Thank you.

**Comment by xopham** [comment, marked as answer]
@gals-ma this is up and coming in v2.2.0 bound to be released within the next 2 weeks. We just need to make some final fixing on an issue we found during testing #347 and it will then work via the standard `helm upgrade`

**Reply by xopham** [reply]
@gals-ma upgrade is now fixed with the latest release. For more information, please check [the docs](https://sse-secure-systems.github.io/connaisseur/v2.2.0/basics/#upgrade).
